src/dialogs/examples/basic/ListCardDialog.ts

```diff
@@ -0,0 +1,52 @@
+import * as builderteams from "botbuilder-teams";
+import * as builder from "botbuilder";
+import * as config from "config";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+// import { ListCard, ListCardItem, ListCardItemType } from "../basic/ListCard";
+import { Strings } from "../../../locale/locale";
+
+export class ListCardDialog extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        let list = Array<builderteams.ListCardItem>();
+
+        // list item1
+        let item1 = new builderteams.ListCardItem();
```

Comment:
Use session, and chain the fluent methods.
new builderteams.ListCardItem(session)
.title(Strings.left_card_item1_title)
...